code/UI/Chat/SportsChatBox.cs

```diff
@@ -131,7 +131,7 @@ public static void Say( string message, bool global = true )
 
 		if ( global )
 			AddChatEntry( To.Everyone, ConsoleSystem.Caller.Name, message, $"avatar:{ConsoleSystem.Caller.PlayerId}", "Global" );
-		else if ( ConsoleSystem.Caller.Components.Get<PartyComponent>() is PartyComponent comp && comp.Party.IsValid() )
+		else if ( ConsoleSystem.Caller.GetPartyComponent() is PartyComponent comp && comp.Party.IsValid() )
```

Comment:
Shouldn't we use GetParty here?

Reply:
it was using the component below this line so i thought it needed it. i changed it now.

Reply:
Awesome.